### ISSUE TITLE
SayAll restarts automatically

### ISSUE THREAD

**Issue by CyrilleB79** [nvaccess/nvda#12255]
### Steps to reproduce:

* Enable "Allow skim reading" in document formatting settings
* Open a webpage, e.g. NVDA user guide.
* Press control key to be sure to stop sayAll
* Press H to go to next title

### Actual behavior:

NVDA reads the next title and then restarts in sayAll.

### Expected behavior:

NVDA reads the title and stops speaking.

#### NVDA installed/portable/running from source:
From source
#### NVDA version:
alpha-22222,2e37433e with the modification of https://github.com/nvaccess/nvda/issues/12160#issuecomment-811031134 to avoid crash.

#### Windows version:
10 1809
#### Does the issue still occur after restarting your computer?
Yes
#### Have you tried any other versions of NVDA? If so, please report their behaviors.
2020.4 is OK
#### If add-ons are disabled, is your problem still occurring?
Not tested.
#### Did you try to run the COM registry fixing tool in NVDA menu / tools?
No

Cc @lukaszgo1

**Comment by lukaszgo1** [comment]
Cannot reproduce with NVDA version alpha-22222,2e37433e in Firefox. Are you sure this is not caused by add-ons?

**Comment by CyrilleB79** [comment]
OK.
I have tested to disable add-on.
Finally, the issue seems due to Speech history add-on (jscholes/nvda-speech-history#16).
Closing this on NVDA side.
